pkg/core/user/user.go

```diff
@@ -0,0 +1,17 @@
+package user
+
+type User struct {
```

Comment:
but probably this package should not be named `user`, maybe `rbac`? Besides `User` it also contains Role and RoleAssignments

Reply:
I kept user in `user` package but moved stuff to pkgcore/rbac